.env

```diff
@@ -0,0 +1,12 @@
+NODE_ENV=development
```

Comment:
`.env` file should be added to `.gitignore`. It should not be added to your commits